src/app/projects/page.tsx

```diff
@@ -141,8 +144,20 @@ export default function Projects() {
     [],
   )
 
+  const hasTierAboveZero = useMemo(() => {
+    return !isLoading && tier > 0 ? true : false
+  }, [tier, isLoading])
+
   return (
     <>
+      {!hasTierAboveZero && (
+        <Link href={"/stake"}>
+          <div className="flex w-full h-10 justify-center items-center bg-brandBlue-100">
+            You are in tier 0, please go to the stake page and raise your tier
```

Comment:
"Your investment tier is too low at the moment. Please increase it by staking $SYS on the stake page"